capnp-rpc/Cargo.toml

```diff
@@ -21,3 +21,9 @@ features = ["std"]
 [dependencies]
 capnp-futures = { version = "0.18.0", path = "../capnp-futures" }
 capnp = {version = "0.18.0", path = "../capnp"}
+
+#[lints]
+#workspace = true
```

Comment:
Why commented out?

Reply:
as above. this should be uncommented as soon as the type complexity lint is addressed.

Would you prefer i remove the commented sections?